settings/DevHome.Settings/Strings/en-us/Resources.resw

```diff
@@ -406,4 +465,22 @@
   <data name="Settings_Accounts_NoProvidersContentDialog_SecondaryButtonText" xml:space="preserve">
     <value>Cancel</value>
   </data>
-</root>
+  <data name="ExperimentalFeature_1_Description" xml:space="preserve">

```

Comment:
Are we guaranteed to have a real experimental feature before 11/14 (code complete for next release)?  We don't want to have temporary strings in a public release.